examples/advanced/main.go

```diff
@@ -36,19 +38,44 @@ import (
 // main is compiled to an exported Wasm function named "_start", called by the
 // Wasm scheduler plugin during initialization.
 func main() {
+	// Instead of using `plugin.Set`, this configures only the interfaces
+	// implemented by the plugin. The Wasm host only calls functions imported,
+	// so this prevents additional overhead.
```

Comment:
Wouldn't it be the same if we use `plugin.Set`? since `plugin.Set` does the interface checks.

Reply:
I see, you're right. When using `plugin.Set`, it also performs an interface check before calling each plugin. Therefore, it seems there is no difference between using `SetPlugin` and `Set`.
I will remove comments here.

Reply:
And then, do we still need to use `SetPlugin` here?